libs/webb-ui-components/src/components/buttons/Button.tsx

```diff
@@ -94,6 +96,18 @@ const Button = forwardRef<HTMLElement, ButtonProps>((props, ref) => {
       )}
     </Component>
   );
+
+  if (disabledTooltip === undefined || !isDisabled) {
```

Comment:
`disabledTooltip !== undefined && isDisabled`

We should keep the default behavior if the button is disabled but not render it with the tooltip.